tests/base/symmetric_tensor_40.cc

```diff
@@ -0,0 +1,19 @@
+#include <fstream>
```

Comment:
this test should be renamed. It does NOT test `symmetric_tensor`, please renamed it to `vectorization_XY`.